### PR TITLE
[bitnami/redis-cluster] perf: add cluster-announce port and bus-port

### PR DESCRIPTION
**Pull request by wojiushixiaobai**
### Description of the change

Add cluster-announce-port and cluster-announce-bus-port to allow users to run redis clusters on a single node.

`REDIS_CLUSTER_ANNOUNCE_IP`
`REDIS_CLUSTER_ANNOUNCE_PORT`
`CLUSTER_ANNOUNCE_BUS_PORT`

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

File: bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/librediscluster.sh [hunk 1]
```diff
@@ -91,6 +91,12 @@ redis_cluster_override_conf() {
         redis_conf_set tls-cluster yes
         redis_conf_set tls-replication yes
     fi
+    if ! is_empty_value "$REDIS_CLUSTER_ANNOUNCE_PORT"; then
+        redis_conf_set "cluster-announce-port" "$REDIS_CLUSTER_ANNOUNCE_PORT"
+    fi
+    if ! is_empty_value "$CLUSTER_ANNOUNCE_BUS_PORT"; then
```
Comment (fmulero) [inline bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/librediscluster.sh:97]:
```suggestion
    if ! is_empty_value "$REDIS_CLUSTER_ANNOUNCE_BUS_PORT"; then
```

File: bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/librediscluster.sh [hunk 2]
```diff
@@ -91,6 +91,12 @@ redis_cluster_override_conf() {
         redis_conf_set tls-cluster yes
         redis_conf_set tls-replication yes
     fi
+    if ! is_empty_value "$REDIS_CLUSTER_ANNOUNCE_PORT"; then
+        redis_conf_set "cluster-announce-port" "$REDIS_CLUSTER_ANNOUNCE_PORT"
+    fi
+    if ! is_empty_value "$CLUSTER_ANNOUNCE_BUS_PORT"; then
```
Comment (fmulero) [inline bitnami/redis-cluster/6.2/debian-11/rootfs/opt/bitnami/scripts/librediscluster.sh:97]:
```suggestion
    if ! is_empty_value "$REDIS_CLUSTER_ANNOUNCE_BUS_PORT"; then
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Hi @wojiushixiaobai Thank you so much for your contribution.

Could you fix the variable names and add them to [redis-cluster-env.sh](https://github.com/bitnami/containers/blob/d89f4850bb03e8ec945da186e0e6364373b9e4a0/bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/redis-cluster-env.sh#L130) files?

**Comment by wojiushixiaobai** [comment]
Sorry, I think I blew it up.
I'll open a new request.

**Pull request closed without merging**